src/db/review_prefs.rs

```diff
@@ -88,28 +145,33 @@ pub async fn upsert_review_prefs(
     db: &tokio_postgres::Client,
     user: User,
     max_assigned_prs: Option<u32>,
+    rotation_mode: RotationMode,
 ) -> anyhow::Result<u64, anyhow::Error> {
     // We need to have the user stored in the DB to have a valid FK link in review_prefs
     record_username(db, user.id, &user.login).await?;
 
     let max_assigned_prs = max_assigned_prs.map(|v| v as i32);
     let query = "
-INSERT INTO review_prefs(user_id, max_assigned_prs)
-VALUES ($1, $2)
+INSERT INTO review_prefs(user_id, max_assigned_prs, rotation_mode)
+VALUES ($1, $2, $3)
 ON CONFLICT (user_id)
 DO UPDATE
-SET max_assigned_prs = excluded.max_assigned_prs";
+SET max_assigned_prs = excluded.max_assigned_prs,
+    rotation_mode = excluded.rotation_mode";
 
     let res = db
-        .execute(query, &[&(user.id as i64), &max_assigned_prs])
+        .execute(
+            query,
+            &[&(user.id as i64), &max_assigned_prs, &rotation_mode],
+        )
         .await
         .context("Error upserting review preferences")?;
     Ok(res)
 }
 
 #[cfg(test)]
 mod tests {
```

Comment:
Unless I'm missing something, we are never checking that the rotation mode is the correct. Could you add test for it.

Reply:
I will definitely add tests, good point. Not sure what do you mean by it being correct, though? It's an enum, it can only be in two states and both are valid.

Reply:
I meant that we should make sure that the updated rotation mode is the one we expect, nothing more.